frameworks/mxnet/get_started_mnist_deploy.ipynb

```diff
@@ -0,0 +1,608 @@
+    "mask = random.sample(range(images.shape[0]), 16)\n",
+    "samples = images[mask]\n",
+    "\n",
+    "# plot the images \n",
+    "fig, axs = plt.subplots(nrows=1, ncols=16, figsize=(16, 1))\n",
+    "\n",
+    "for i, splt in enumerate(axs):\n",
+    "    splt.imshow(samples[i])\n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, let us use the model to infer the samples one-by-one. This is the typical use case\n",
+    "for an online application."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# convert to float and normalize normalize the input \n",
+    "\n",
+    "def normalize(x, axis):\n",
+    "    eps = np.finfo(float).eps\n",
+    "    mean = np.mean(x, axis=axis, keepdims=True)\n",
+    "    # avoid division by zero\n",
+    "    std = np.std(x, axis=axis, keepdims=True) + eps\n",
+    "    return (x - mean) / std\n",
+    "\n",
+    "\n",
+    "samples = normalize(samples.astype(np.float32), axis=(1, 2)) # mean 0; std 1\n",
+    "\n",
+    "res = []\n",
+    "for img in samples:\n",
+    "    data = {\n",
+    "        \"inputs\": img.flatten().tolist()\n",
+    "    }\n",
+    "    res.append(predictor.predict(data)[0])"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Predictions: \", *map(int, res))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since in `transform_fn`, the parsed numpy array could have take on any value for its batch\n",
+    "dimension, we can send the entire `samples` at once and let the model do a batch inference."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data = {\n",
+    "    \"inputs\": samples.tolist()\n",
+    "}\n",
+    "res = predictor.predict(data)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Predictions: \", *map(int, res))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test and debug the entry point before deployment\n",
+    "\n",
+    "When deploying a model to a SageMaker endpoint, it is a good practice to test the entry \n",
+    "point. The following snippet shows you how you can test and debug the `model_fn` and \n",
+    "`transform_fn` you implemented in the entry point for the inference image.  "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pygmentize code/test_inference.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The `test` function simulates how the inference container works. It pulls the model\n",
+    "artifact loads the model into \n",
```

Comment:
lot of extra line breaks here that aren't needed
also, "... artifact **and** loads..."

Reply:
looks like I do have a strong  preference to line breaks:) removed all of them